sdk/containerregistry/Azure.Containers.ContainerRegistry/samples/Sample03a_SetImageProperties.md

```diff
@@ -0,0 +1,38 @@
+# Azure.Containers.ContainerRegistry Samples - Set Image Properties (sync)
+
+A common use case for Azure Container Registries is to set an image's properties so it can't be overwritten during a lengthy deployment.
+
+The following sample assumes the registry `myacr.azurecr.io` has a repository `hello-world` where one image has two tags: `v1` and `latest`.
+
+```C# Snippet:ContainerRegistry_Tests_Samples_SetArtifactProperties
+// Get the service endpoint from the environment
+Uri endpoint = new Uri(Environment.GetEnvironmentVariable("REGISTRY_ENDPOINT"));
+
+// Create a new ContainerRegistryClient and RegistryArtifact to access image operations
+ContainerRegistryClient client = new ContainerRegistryClient(endpoint, new DefaultAzureCredential());
+RegistryArtifact image = client.GetArtifact("library/hello-world", "v1");
+
+// Set permissions on the v1 image's "latest" tag
+image.UpdateTagProperties("latest", new ArtifactTagProperties()
+{
+    CanWrite = false,
+    CanDelete = false
+});
+```
+
+If someone were to push an update to `myacr.azurecr.io\hello-world:v1`, it would fail.
```

Comment:
The example above seems to be setting permissions on "latest" but this comment mentions the "v1" tag. It seems like they should both be the same, one way or the other. Based on the following text about being cautious, it seems like the above example should all be about "v1"

Reply:
Let me think about how to clarify this. I wanted to make the point that you can lock one tag, but if another tag is unlocked the image can still be updated.